mangum/adapter.py

```diff
@@ -112,15 +113,15 @@ def handle_ws(self, event: dict, context: dict) -> dict:
             # The initial connect event. Parse and store the scope for the connection
             # in DynamoDB to be retrieved in subsequent message events for this request.
             server, client = get_server_and_client(event)
-            headers = event["headers"]
+            headers = event.get("headers", dict())
```

Comment:
We want to duplicate the header definition in the HTTP handler for WebSockets as well.

Reply:
I swapped this line for a dict literal as well.  Was that the change you were looking for?